Dockerfile.temporal

```diff
@@ -0,0 +1,23 @@
+FROM temporalio/auto-setup:1.20
```

Comment:
it would be better to pin to a specific minor version (e.g., `1.20.0`) for better reproducibility